doc/nrf/releases/release-notes-latest.rst

```diff
@@ -0,0 +1,31 @@
+.. _ncs_release_notes_latest:
+
+Changes in |NCS| v1.3.99
+########################
+
+The most relevant changes that are present on the master branch of the |NCS|, as compared to the latest release, are tracked in this file.
+Note that this file is a work in progress and might not cover all relevant changes.
+
+
+Change log
+**********
+
+See the `master branch section in the change log`_ for a list of the most important changes.
+
+sdk-nrfxlib
+===========
+
+See the change log for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for the most current information.
+
+sdk-zephyr
+==========
+
+The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``ac117c8d``, plus some |NCS| specific additions.
+
+The following list summarizes the most important changes from upstream Zephyr:
```

Comment:
The distinction between this bullet list and the second one took me a couple of tries to understand, because "changes from upstream zephyr" could be read as either of these:

1. divergence in NCS's fork of zephyr from the upstream zephyr repository
2. changes in NCS's fork of zephyr in the master branch since the most recent release

On first read, I assumed it was 2., but actually it seems to be 1. after reading the below on line 29. Since the list is likely to grow quite long, I think it is worth it to clarify this here, since line 29 will be line 400 at some point, I am guessing, and will not be visible on screen at the same time.

Is there some way to reword this to be more clear?

Reply:
Updated to say "The following list summarizes the most important **commits** from upstream Zephyr:".

Reply:
> Updated to say "The following list summarizes the most important **commits** from upstream Zephyr:".

That is still a bit confusing. How about "The following list summarizes the most important changes inherited from upstream Zephyr:" ?

Reply:
Either works. Updated.